bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImplTest.java

```diff
@@ -305,4 +305,11 @@ private FormContainer getFormContainerWithLocaleUnderTest(String resourcePath) t
         context.currentResource().adaptTo(FormContainer.class);
         return request.adaptTo(FormContainer.class);
     }
+
+    @Test
+    void testGetName() throws Exception {
+        FormContainer formContainer = Utils.getComponentUnderTest(PATH_FORM_1, FormContainer.class, context);
+        assertNull(formContainer.getName());
+        // assertEquals("L2NvbnRlbnQvZm9ybXMvYWYvZGVtbw==", formContainer.getId());
```

Comment:
you may remove the unwanted commented code, please.

Reply:
> you may remove the unwanted commented code, please.

done!